lib/puppet/network/authconfig.rb

```diff
@@ -78,6 +78,12 @@ def mk_acl(acl)
     # raise an Puppet::Network::AuthorizedError if the request
     # is denied.
     def check_authorization(method, path, params)
+      if Puppet.settings[:bypass_authorization] == true
```

Comment:
Can you just do:

```
if Puppet[:bypass_authorization]
```

since the setting is defined to be a `:boolean`, you'll either get back ruby `true` or `false` values.